google-cloud-error_reporting/lib/google/cloud/error_reporting.rb

```diff
@@ -62,8 +62,7 @@ module ErrorReporting
       #   * `https://www.googleapis.com/auth/cloud-platform`
       #
       # @param [Integer] timeout Default timeout to use in requests. Optional.
-      # @param [Hash] client_config A hash of values to override the default
-      #   behavior of the API client. Optional.
+      # @param [Hash] client_config Unused.
```

Comment:
Do you want to explicitly label this as deprecated?

Reply:
Yeah, that's a good idea, we should.

Reply:
Done